TypeCobol/Compiler/AntlrUtils/CobolErrorStrategy.cs

```diff
@@ -337,9 +293,11 @@ public class CodeElementErrorStrategy : CobolErrorStrategy
         /// </summary>
         protected override bool ErrorStrategyShouldNotConsumeThisToken(Token lastConsumedToken, Token nextToken)
         {
+            if (nextToken.TokenType == TokenType.SQL_FROM) return false;
+
             return nextToken.TokenFamily == TokenFamily.CompilerDirectiveStartingKeyword ||
```

Comment:
We can also delete `nextToken.TokenFamily == TokenFamily.CompilerDirectiveStartingKeyword` I think.

Reply:
Definitely ! Forgot that after writing it as a test.